### ISSUE TITLE
Refactoring Essentials use in Visual Studio 2017

### ISSUE THREAD

**Issue by PhilSargeant** [icsharpcode/RefactoringEssentials#321]
Hello,
Please excuse my ignorance but I’m having trouble using (or at least knowing how to use) Refactoring Essentials on my Visual Studio 2017.

I have no problems with Refactoring Essentials on my Visual Studio 2015. I simply highlight the code,  right click the mouse and then select  “Convert to …. And copy to clip board” – no problem.

If I do the same thing in Visual Studio 2017, I do not see any mention of “Convert to ….”  The only thing that I see is the yellow bulb to the left and if I select this the only thing I see is “Change Signature”.

I have downloaded a number of Extensions and see them in either “View -> Windows” or in Tools, but I see nothing with respect to Refactoring Essentials.

Any help would be much appreciated.
Regards
Phil Sargeant

**Comment by GrahamTheCoder** [comment]
Thanks for the bug report. Apologies for the lack of communication, as of today I'm looking after the project in its new home, so I've duplicated the issue which can now be closed here. 
https://github.com/icsharpcode/CodeConverter/issues/17

I can't reproduce the issue. I see menu items, but then have problems loading some assemblies (https://github.com/icsharpcode/CodeConverter/issues/13). If anyone wishes to provide an [Activity Log](https://blogs.msdn.microsoft.com/visualstudio/2010/02/24/troubleshooting-extensions-with-the-activity-log/) of the failure, it might help diagnosis.

**Comment by PhilSargeant** [comment]
I submitted this issue on July 8 2017. At that time the convert functionality of Refactoring Essentials in Visual Studio was definitely NOT working.

At some point towards the end of 2017 I installed an updated version of Refactoring Essentials and low and behold the conversion functionality appeared.

“My bad” for not having reflected this on GitHub.

Whatever was done solved the problem. Well done!

Cheers
Phil Sargeant
